src/components/Home.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import Social from './Social'
+import People from './People'
+import TimeCard from './TimeCard'
+import content from '../content'
+import Header from './Header'
+import './Home.modules.css'
+
+const Home = () => (
+  <div>
```

Comment:
If the `<div` tag doesn't bring any value, you should use a react fragment instead.